### PR TITLE
Test enhancement

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log
- It's about the test enhancement.
- Add `php-7.3` test during Travis CI build.
- Define multiple scripts in `composer.json`.
- Set the current coding style check paths and skipping warning setting value.

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 >
     <testsuites>
         <testsuite name="tests">
-            <directory>tests/</directory>
+            <directory suffix="Test.php">test/</directory>
```
Comment (peter279k) [inline phpunit.xml:12]: Set current target test directory.

File: test/CollectionTest.php [hunk 2]
```diff
@@ -1,6 +1,11 @@
 <?php
 
-class CollectionTest extends \PHPUnit\Framework\TestCase
+namespace Dimsh\Models\Collections\Tests;
```
Comment (peter279k) [inline test/CollectionTest.php:3]: Add ` Dimsh\Models\Collections\Tests;` namespace for every test classes.

File: composer.json [hunk 3]
```diff
@@ -31,18 +31,23 @@
...
+      "@check-style",
+      "@test-coverage"
     ]
   },
   "scripts-descriptions": {
-    "test": "Run all tests and quality checks"
+    "test": "Run unit tests",
+    "test-coverage": "Run unit tests and generate coverage clover",
+    "check-style": "Check specific coding style",
+    "check-all": "Run @check-style and @test-coverage scripts"
```
Comment (peter279k) [inline composer.json:51]: Defining multiple composer scripts and they also have the script descriptions.

File: composer.json [hunk 4]
```diff
@@ -31,18 +31,23 @@
     }
   },
   "autoload-dev": {
-    "classmap": [
-      "test/TestBasicObject.php",
-      "test/TestBasicObjectsList.php"
-    ]
+    "psr-4": {
+      "Dimsh\\Models\\Collections\\Tests\\": "test/"
```
Comment (peter279k) [inline composer.json:35]: Add the `PSR-4` namespace for test classes.

**Review by idimsh — APPROVED** [review]
Thanks for the contrib

**Pull request merged**